src/commcare_cloud/ansible/roles/elasticsearch/tasks/rolling_restart.yml

```diff
@@ -6,26 +6,46 @@
   retries: 20
   delay: 3
   changed_when: result.stdout.find('"acknowledged":true') != -1
+  tags: action_stop
+
+- debug: msg="{{ result }}"
+  tags: action_stop
+
 
 - name: stop node
   become: true
   service: name=elasticsearch state=stopped
+  tags: action_stop
 
 - name: wait for a few seconds for ES to stop
   pause: seconds=10
+  tags: action_stop
+
+- name: get es instances to kill
+  shell: "ps aux | pgrep 'elasticsearc[h]' | awk '{print $2}'"
+  register: es_instances
+  tags: action_stop
+
+- name: kill elasticsearch instances
+  shell: "pkill {{item}}"
+  with_items: es_instances
```

Comment:
Does this work? I would have though that you would have to use `es_instances.stdout.splitlines()` or something

Reply:
https://github.com/dimagi/commcare-cloud/pull/1990/commits/9328828602a703ee1aee357628c85c6e886c9433